cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/TimeStampFromSource.java

```diff
@@ -0,0 +1,38 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.biobank.model.cristin.CristinTimeStampFromSource;
+
+import java.time.Instant;
+
+public class TimeStampFromSource {
```

Comment:
You call the class `TimeStampFromSource`, so fields can just be `timeStamp` and `source`. I would not have use camel-case for timeStamp, just timestamp.